parm/FV3LAM_wflow.xml

```diff
@@ -30,42 +31,43 @@ Parameters needed by the job scheduler.
 <!--
 Workflow task names.
 -->
-<!ENTITY MAKE_GRID_TN            "{{ make_grid_tn }}">
-<!ENTITY MAKE_OROG_TN            "{{ make_orog_tn }}">
-<!ENTITY MAKE_SFC_CLIMO_TN       "{{ make_sfc_climo_tn }}">
-<!ENTITY GET_EXTRN_ICS_TN        "{{ get_extrn_ics_tn }}">
-<!ENTITY GET_EXTRN_LBCS_TN       "{{ get_extrn_lbcs_tn }}">
-<!ENTITY GET_GEFS_LBCS_TN        "{{ get_gefs_lbcs_tn }}">
-<!ENTITY GET_EXTRN_LBCS_LONG_TN  "{{ get_extrn_lbcs_long_tn }}">
-<!ENTITY MAKE_ICS_TN             "{{ make_ics_tn }}">
-<!ENTITY MAKE_LBCS_TN            "{{ make_lbcs_tn }}">
-<!ENTITY RUN_FCST_TN             "{{ run_fcst_tn }}">
-<!ENTITY RUN_POST_TN             "{{ run_post_tn }}">
-<!ENTITY RUN_PRDGEN_TN           "{{ run_prdgen_tn }}">
-<!ENTITY RUN_BUFRSND_TN          "{{ run_bufrsnd_tn }}">
-<!ENTITY ANAL_GSI_TN             "{{ anal_gsi }}">
-<!ENTITY ANAL_GSIDIAG_TN         "{{ anal_gsidiag }}">
-<!ENTITY ANAL_SD_GSI_TN          "{{ anal_sd_gsi }}">
-<!ENTITY POSTANAL_TN             "{{ post_anal }}">
-<!ENTITY OBSERVER_GSI_ENSMEAN_TN "{{ observer_gsi_ensmean }}">
-<!ENTITY OBSERVER_GSI_TN         "{{ observer_gsi }}">
-<!ENTITY PREP_CYC_SPINUP_TN      "{{ prep_cyc_spinup }}">
-<!ENTITY PREP_CYC_PROD_TN        "{{ prep_cyc_prod }}">
-<!ENTITY PREP_CYC_TN             "{{ prep_cyc }}">
-<!ENTITY CALC_ENSMEAN_TN         "{{ calc_ensmean }}">
-<!ENTITY PROCESS_RADAR_REF_TN    "{{ process_radarref }}">
-<!ENTITY PROCESS_LIGHTNING_TN    "{{ process_lightning }}">
-<!ENTITY PROCESS_GLMFED_TN       "{{ process_glmfed }}">
-<!ENTITY PROCESS_BUFR_TN         "{{ process_bufr }}">
-<!ENTITY RADAR_REFL2TTEN_TN      "{{ radar_refl2tten }}">
-<!ENTITY CLDANL_NONVAR_TN        "{{ cldanl_nonvar }}">
-<!ENTITY SAVE_RESTART_TN         "{{ save_restart }}">
-<!ENTITY SAVE_DA_OUTPUT_TN       "{{ save_da_output }}">
-<!ENTITY JEDI_ENVAR_IODA_TN      "{{ jedi_envar_ioda }}">
-<!ENTITY PROCESS_SMOKE_TN        "{{ process_smoke }}">
-<!ENTITY PROCESS_PM_TN           "{{ process_pm }}">
-<!ENTITY ADD_AEROSOL_TN          "{{ add_aerosol_tn }}">
 
+<!ENTITY MAKE_GRID_TN      "{{ make_grid_tn }}">
```

Comment:
@EdwardColon-NOAA, you removed the latest update. Please revert the change. In the latest PR, there variables are aligned but your PR gets them back to the older version.